designsafe/static/scripts/data-depot/components/projects/pipeline-publish/pipeline-publish.component.js

```diff
@@ -84,6 +86,12 @@ class PipelinePublishCtrl {
             });
         }
         this.publication = publication;
+        this.PublishedService.getPublished(this.project.value.projectId).then((resp) => {
+            let data = resp.data;
+            if (data.project) {
+                this.existingPub = true;
```

Comment:
just noticed that this is a potential race condition since `this.PublishedService.getPublished` is dispatched as an asynchronous action. Perhaps set `this.existingPub = null` at the top of the `onInit` instead of `false`, and flag it `true` or `false` here. Then in the template, add a loading icon, or just disable the submit button until `this.existingPub !== null`

Reply:
I see what you mean, I think that makes a little more sense. I'll get those changes added.